AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_entity_utils.py

```diff
@@ -468,7 +468,7 @@ def add_component(self, component_name: str) -> EditorComponent:
         :return: Component object of newly added component.
         """
         component = self.add_components([component_name])[0]
-        PrefabUtils.wait_for_propagation()
+        PrefabWaiter.wait_for_propagation()
```

Comment:
seems like this should move into add_components since this simply calls that with a list of one component.
if the wait_for_propogation only exists for single add_component() then if a list of components is added no wait_for_propogation is called

Reply:
Done.